### PR TITLE
isc: add min/max for 8-byte double arrays demo

### LINKED ISSUE #1213: Extend ISC demo with another min/max example
Extend ISC demo with another min/max example, for the case when the data is stored as 8-byte double numbers arrays.

### PR DESCRIPTION
**Pull request by andriytk**
# Problem Statement
Extend ISC demo with another min/max example, for the case when the data is stored as 8-byte double numbers arrays.

# Design
The design is based on the existing min/max example for the string numbers, but it's much simpler now. Because we don't have to deal with the edge cases (when numbers may be split between several units).

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [x] Side effects on other features (deployment/upgrade)
- [x] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

Closes #1213.

### CODE REVIEW

File: iscservice/demo/libdemo.c [hunk 1]
```diff
@@ -170,6 +173,46 @@ int compute_minmax(enum op op, struct m0_isc_comp_private *pdata,
...
+	m0_bcount_t       len;
+	struct mm_result  res = {};
+	struct m0_buf     buf = M0_BUF_INIT0;
+
+	len = m0_isc_io_res((struct m0_stob_io *)pdata->icp_data, &(char*)p);
+	if (len < 0) {
+		*rc = M0_ERR_INFO((int)len, "failed to read data");
+		return M0_FSO_AGAIN;
+	}
+
```
Comment (huanghua78) [inline iscservice/demo/libdemo.c:190]: Do you like:
M0_ASSERT(len % 8 == 0);
?
It is obviously true, but this ensures that no boundary cutting happens.

Comment (andriytk) [reply iscservice/demo/libdemo.c:190]: Good idea, added. Thanks!

**Review by huanghua78 — APPROVED** [review]

**Review by siningwuseagate — APPROVED** [review]

**Comment by mehjoshi** [comment]
retest this please

File: iscservice/demo/iscdemo.c [hunk 2]
```diff
@@ -118,6 +128,8 @@ static int input_prepare(struct m0_buf *buf, struct m0_fid *comp_fid,
 		return ping_input_prepare(buf, comp_fid, reply_len, type);
 	case ICT_MIN:
 	case ICT_MAX:
+	case ICT_MIN2
```
Comment (mehjoshi) [inline iscservice/demo/iscdemo.c:131]: Missing ':'

**Comment by mehjoshi** [comment]
@andriytk , build is failing with this change due to missing ':' in the switch-case.

File: iscservice/demo/iscdemo.c (hunk 2 above)
Comment (andriytk) [reply iscservice/demo/iscdemo.c]: Fixed.

**Review by mehjoshi — APPROVED** [review]

**Pull request merged**